packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -5,6 +5,7 @@
 import pytest
 import retrying
 
+from prometheus_client.parser import text_string_to_metric_families
```

Comment:
`prometheus_client` is a third-party package, so it should be grouped above with `pytest` and `retrying`.